mimic/model/valkyrie_objects.py

```diff
@@ -0,0 +1,140 @@
+"""
+Model objects for the Valkyrie mimic.
+"""
+from characteristic import attributes, Attribute
+from uuid import uuid4
+from json import loads, dumps
+
+from mimic.util.helper import random_hex_generator
+
+class AccountContactPermission(object):
+    """
+    An intersection object representing a certain contact's permissions on a certain account's items
+    """
+
+    permission_type_map = {
```

Comment:
I don't actually even have authoritative docs on the permission types at this point, I just inferred them from sample outputs in the docs.  I'm awaiting responses from the Valkyrie team on special permission types (I believe) they're adding for us.  The list may turn out to have gaps in it, but I'll keep the suggestion in mind.